Sources/NIOHPACK/HPACKHeader.swift

```diff
@@ -486,14 +528,16 @@ extension String.UTF8View {
             }
     }
 
+    @inlinable
     @inline(never)
-    private func _compareCaseInsensitiveASCIIBytesSlowPath(to other: String.UTF8View) -> Bool {
+    func _compareCaseInsensitiveASCIIBytesSlowPath(to other: String.UTF8View) -> Bool {
```

Comment:
This looks odd - inalienable but never?

Reply:
Yup. This is a logically sensible stanza. `@inlinable` means the definition is available to the compiler when compiling the module using the code. This allows, for example, specialization. But the method may not be _inlined_. In this case the primary use is to let the compiler see through the call and observe that we never write to any memory here.